src/styles/utils/_variables.scss

```
@@ -0,0 +1,32 @@
+// stars
+$star-size: 16px;
+$star-indent: $star-size / 4;
+
+//header
+$header-padding: 0 50px;
+
+$image-size: 40px;
+$image-margin: $image-size * 0.25;
+
+$nav-link--height: 60px;
+$nav-item--indent: $nav-link--height / 3;
+
+//card
+$card-width: 200px;
+$card-paddings: 0 $card-width * 0.08;
+
+$image-width: 160px;
+$image-height: 134px;
+$image-padding--left: ($image-width - $image-height) * 0.11;
+$image-padding--top: ($image-width - $image-height) * 1.23;
+$image-padding--bottom: ($image-width - $image-height) * 1.54;
+
+//container
+$container-magrins--vert: $card-width * 0.25 0;
+$container__card-margin--right: $card-width * 0.24;
+$container__card-margin--bottom: $card-width * 0.23;
+
+//textcolor
+$color-1: #060b35;
```

Rename all vars by the example:
```suggestion
$main-accent-color: #060b35;
```